src/Model/Entity/Asset.php

```diff
@@ -313,7 +315,7 @@ public function copyToWebroot(?string $filename = null, string $path = 'files',
 
         $prefix = null;
         if (empty($config['no_prefix'])) {
-            $prefix = Strings::substring(md5($this->modified->toDateTimeString()), 0, 3) . '-';
+            $prefix = Strings::substring(md5($this->modified?->toDateTimeString() ?? ''), 0, 3) . '-';
```

Comment:
this should be looked into.
As said in another issue `modified` shouldn't be nullable in the first place.